content/usgs-packages/sbtools_sbitem.Rmd

```diff
@@ -0,0 +1,82 @@
+---
+title: "sbtools - sbitem"
+date: "9999-07-20"
+author: "Lindsay R. Carr"
+slug: "sbtools-sbitem"
+image: "img/main/intro-icons-300px/r-logo.png"
+output: USGSmarkdowntemplates::hugoTraining
+parent: Introduction to USGS R Packages
+weight: 2
+draft: true
+---
+
+```{r setup, include=FALSE, warning=FALSE, message=FALSE}
+library(knitr)
+
+knit_hooks$set(plot=function(x, options) {
+  sprintf("<img src='../%s%s-%d.%s'/ title='%s'/>", 
+          options$fig.path, options$label, options$fig.cur, options$fig.ext, options$fig.cap)
+
+})
+
+opts_chunk$set(
+  echo=TRUE,
+  fig.path="static/sbtools-sbitem/",
+  fig.width = 6,
+  fig.height = 6,
+  fig.cap = "TODO"
+)
+
+set.seed(1)
+```
+
+"items" are the fundamental unit of data that are available through ScienceBase, and are necessary to understand for using `sbtools`. This lesson will introduce ScienceBase items and the R equivalent, `sbitem`. 
+
+## What is a ScienceBase "item"?
+
+A ScienceBase "item" is any digital object available through ScienceBase. Items can contain files, contain and display metadata, or contain other items to create a hierarchical item/folder structure. Items all follow the same type of metadata model, so they have a structured format. Visit the [ScienceBase Items help page](https://www.sciencebase.gov/about/content/sciencebase-items) for more information, and look at the table below for some examples of items.
```

Comment:
this paragraph is great - captures a lot right up top. maybe change "any digital object" to "the fundamental digital object" so nobody goes thinking that a file also counts as an item. i know you just said "fundamental" in line 33...not sure how to avoid that, because i think it applies to both sentences...maybe line 33 could say '"Items" are the objects in which ScienceBase stores and organizes data' or something?